consul_exporter.go

```diff
@@ -98,10 +102,12 @@ var (
 	queryOptions = consul_api.QueryOptions{}
 )
 
-type promHTTPLogger struct{}
+type promHTTPLogger struct {
+	logger log.Logger
+}
 
 func (l promHTTPLogger) Println(v ...interface{}) {
-	log.Error(v...)
+	level.Error(l.logger).Log("msg", fmt.Sprintf("%v", v))
```

Comment:
Sorry my previous comment was confusing, it should be `level.Error(l.logger).Log("msg", fmt.Sprint(v))`